src/polysolve/nonlinear/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ set(SOURCES
 	Solver.cpp
 	Problem.hpp
 	Problem.cpp
+	PoststepData.hpp
```

Comment:
Minor style comment, but since the method is called `post_step` can this be changed to `PostStepData`?